docs/examples/private_attributes_underscore_attrs_are_private.py

```diff
@@ -0,0 +1,16 @@
+from typing import ClassVar
+
+from pydantic import BaseModel
+
+
+class Model(BaseModel):
+    __class_var__: ClassVar[str] = 'class var value'
+    __private_attr__: str = 'private attr value'
+
+    class Config:
+        underscore_attrs_are_private = True
```

Comment:
I this needs documenting in `model_config.md`. Also we need to be clear exactly what form names need to be in.

It looks from this like if you use `PrivateAttr` you can use `_whatever`, but with `underscore_attrs_are_private` you have to use `__whatever__`, but reading the code, I think that's not the case.

Reply:
actually you've written it clear in the copy, but still better to use the same format in both code examples.

Is there anything special about `__whatever__` or is it just that it starts with an underscore, like `_whatever`?

Reply:
Since I think according to some PEP we're not supposed to use dunder methods, unless they're official to python, maybe leave them out of the example and just mention them in the text?